zebra-network/src/peer/connection.rs

```diff
@@ -2,12 +2,12 @@
 //!
 //! Maps the external Zcash/Bitcoin protocol to Zebra's internal request/response
 //! protocol.
-//!
-//! This module contains a lot of undocumented state, assumptions and invariants.
-//! And it's unclear if these assumptions match the `zcashd` implementation.
-//! It should be refactored into a cleaner set of request/response pairs (#1515).
 
```

Comment:
We've dealt with the undocumented state, but there are still quite a few undocumented assumptions and invariants. See #1515 for details.

```suggestion
//!
//! This module contains a lot of undocumented assumptions about the Zcash
//! network protocol. We don't know if these assumptions match the `zcashd`
//! implementation. It should be refactored into a cleaner set of
//! request/response pairs (#1515).

```

Reply:
This change shouldn't block merging.

Feel free to do it later, or commit and then admin-merge.